### PR TITLE
fix(@angular/cli): Fixing aliases for blueprint help

### PR DESCRIPTION
**Pull request by sumitarora**
Fixes: https://github.com/angular/angular-cli/issues/5289

### CODE REVIEW

File: packages/@angular/cli/commands/generate.ts [hunk 1]
```diff
@@ -61,7 +61,8 @@ const aliasMap: { [alias: string]: string } = {
   'm': 'module',
   'p': 'pipe',
   'r': 'route',
-  's': 'service'
+  's': 'service',
+  'g': 'guard'
```
Comment (johnpapa) [inline packages/@angular/cli/commands/generate.ts]: i dont think g is an alias for guard. didnt work for me, at least

Comment (sumitarora) [inline packages/@angular/cli/commands/generate.ts]: Referenced this one: https://github.com/angular/angular-cli/pull/5339

Comment (Brocco) [inline packages/@angular/cli/commands/generate.ts]: I added it in PR #5339

Comment (johnpapa) [inline packages/@angular/cli/commands/generate.ts]: cool. so it *will* exist in RC 2 :)

**Pull request merged**